doc/publishing-api-syntactic-usage.md

```diff
@@ -90,7 +90,7 @@ Requests to update an existing draft content item:
   - `content_id` the primary identifier for the content to publish.
   - `type` the type of unpublishing to create/perform.
 
-### Other request params:
+### Optional request params:
```

Comment:
I have not thought through this fully yet, but at the moment I'm just putting the parameters in the "Optional" section if there exists a circumstance in which they are not required. I think this is reasonably consistent with the other parts of this file.